src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationBambooBitbucketJiraTest.java

```diff
@@ -547,6 +547,14 @@ void createProgrammingExercise_notIncluded_invalidBonusPoints_badRequest() throw
         programmingExerciseIntegrationTestService.createProgrammingExercise_notIncluded_invalidBonusPoints_badRequest();
     }
 
+    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
+    @ParameterizedTest(name = "{displayName} [{index}] {argumentsWithNames}")
+    // It should fail for all ProgrammingExercises except Java and Kotlin
```

Comment:
"fail" sounds to me like the test case should fail, not the server request. (same for other tests below)
```suggestion
    // It should return a bad request error for all ProgrammingExercises except Java and Kotlin
```